src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -243,40 +255,48 @@ public void onTraversalStart() {
         if (outputType == CommonCode.OutputType.BQ) {
             loadStatus = new LoadStatus(projectID, datasetName, loadStatusTableName);
 
-            LoadStatus.LoadState state = loadStatus.getSampleLoadState(sampleId);
+            LoadStatus.LoadState state = loadStatus.getSampleLoadState(sampleId, loadHeadersOnly);
             if (state == LoadStatus.LoadState.COMPLETE) {
                 logger.info("Sample id " + sampleId + " was detected as already loaded, exiting successfully.");
                 System.exit(0);
+            } else if (loadHeadersOnly && state == LoadStatus.LoadState.HEADERS_WRITTEN) {
+                logger.info("Sample id " + sampleId + " has already had its headers written, exiting successfully.");
+                System.exit(0);
             } else if (state == LoadStatus.LoadState.PARTIAL) {
-                if (enableReferenceRanges) {
+                if (enableReferenceRanges && !loadHeadersOnly) {
```

Comment:
This and the following two changes are fine, but it feels like this entire block can be simplified by just wrapping it all in a  `if (!loadHeadersOnly) { ... }` block to me.  I'm not sure what the value is in adding that logic to every individual condition in this stretch